lib/influxdb2/client/point.rb

```diff
@@ -138,6 +138,8 @@ def to_line_protocol
     def _escape_tags
       return if @tags.nil?
 
+      @tags = @tags.each_with_object({}) { |(k, v), tag| tag[k.to_s] = v }
+
       @tags.sort.to_h.map do |k, v|
```

Comment:
Please, avoid double iteration by:

```suggestion
      @tags.sort_by{|x, _| x.to_s}.to_h.map do |k, v|
```